src/main/resources/swagger.api/remote-storage.yaml

```diff
@@ -338,13 +338,13 @@ paths:
           schema:
             type: string
   /pub-sub-handlers/moved-event:
```

Comment:
Maybe we should change endpoint name for handling events according new logic (also in MessagingDescriptor.json)